app/models/course/requirement.rb

```diff
@@ -0,0 +1,4 @@
+class Course::Requirement < ActiveRecord::Base
+  actable
+  belongs_to :has_requirement, polymorphic: true # naming is hard
```

Comment:
the "xxx-able" convention is good, but from experience it gives me completely meaningless names. what does 'requirable' mean? 'taggable' makes sense, but what kind of tags?